contracts/introspection/ERC165Checker.sol

```diff
@@ -13,39 +13,60 @@ library ERC165Checker {
   // As per the EIP-165 spec, ERC165_ID == bytes4(keccak256('supportsInterface(bytes4)')
   bytes4 private constant InterfaceId_ERC165 = 0x01ffc9a7;
 
+  /**
+   * @notice Query if a contract implements an interface, does not check ERC165 support
+   * @param _address The address of the contract to query for support of an interface
+   * @param _interfaceId The interface identifier, as specified in ERC-165
+   * @return true if the contract at _address indicates support of the interface with
+   * identifier _interfaceId, false otherwise
+   * @dev Assumes that _address contains a contract that supports ERC165, otherwise
+   *      the behavior of this method is undefined. This precondition can be checked 
```

Comment:
@nventuro what's our stance on indentation within comments? personally I prefer it, but I remember it being deemed unnecessary as a part of one of the other PRs?

Reply:
tbh I have no idea on whether or not they affect docgen. I'd remove all indentation for consistency, and tackle it in a separate PR if needed.